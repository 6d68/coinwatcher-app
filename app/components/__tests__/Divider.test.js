import React from 'react';
import Divider from '../divider';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<Divider />).toJSON();
  expect(rendered).toMatchSnapshot();
});
