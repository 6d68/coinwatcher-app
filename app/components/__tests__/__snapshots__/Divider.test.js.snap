// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "backgroundColor": "grey",
      "height": 0.5,
    }
  }
/>
`;
